src/layerelement.cpp

```diff
@@ -1153,6 +1153,11 @@ int LayerElement::AdjustLayers(FunctorParams *functorParams)
                 }
                 else if (abs(previousNote->GetDrawingLoc() - params->m_currentNote->GetDrawingLoc()) > 1)
                     continue;
+                // check if the next note is simultaneous and put in with adjustable space
```

Comment:
I am not sure how this works. A note will always have a staff ancestor. When do you expect this not to be true? What is the logic of the change?
Also, you do not need to do a dynamic_cast to compare with a NULL.  (Please change `nullptr` to `NULL` for consistency)

Reply:
Sorry, I didn't know about staff ancestors. I removed this check.